cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -834,15 +823,15 @@ public KafkaBrokerConfigurationBuilder withUserConfiguration(KafkaConfiguration
             // Configure the configuration providers => we have to inject the Strimzi ones
             configProviders(userConfig);
 
-            // Handle all permutations for metric.reporters
+            // Handle all combinations of metric.reporters
             String metricReporters = userConfig.getConfigOption(KAFKA_METRIC_REPORTERS_CONFIG_FIELD);
 
             // If the injectCcMetricsReporter / injectStrimziMetricsReporter flag is set to true, it is appended to the list of metric reporters
             if (injectCcMetricsReporter) {
                 metricReporters = appendMetricReporter(metricReporters, CruiseControlMetricsReporter.CRUISE_CONTROL_METRIC_REPORTER);
             }
             if (injectStrimziMetricsReporter) {
-                metricReporters = appendMetricReporter(metricReporters, STRIMZI_METRIC_REPORTER);
+                metricReporters = appendMetricReporter(metricReporters, KAFKA_PROMETHEUS_METRICS_REPORTER);
```

Comment:
Let's avoid static imports, like for CC. It should be `StrimziReporterMetricsModel.KAFKA_PROMETHEUS_METRICS_REPORTER` and `KafkaCluster.KAFKA_METRIC_REPORTERS_CONFIG_FIELD`.